app/views/draws/oversubscription.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1><%= @draw.name %> - Oversubscription handling</h1>
+<div class="group-report">
+  <%= render 'group_report', sizes: @draw_sizes, groups: @groups_by_size, actions_partial: 'groups/actions', path: oversub_draw_path(@draw) %>
+</div>
+<% if policy(@draw).start_lottery? && @proceeding_to_lottery %>
+  <p><%= link_to 'Really proceed to lottery', start_lottery_draw_path(@draw), method: :patch, class: 'button alert',
+                 **with_tooltip(text: 'Clicking this button will proceed to the lottery phase, even if there are still oversubscribed suite sizes!',
+                                class_override: 'button alert') %></p>
```

Comment:
it's a bit confusing to have this button disappear after oversubscription is resolved, but it makes sense because there will be ungrouped students after disbanding groups to resolve oversubscription. 

not really sure how we should fix this